logging/log_collection_forwarding/log-forwarding.adoc

```diff
@@ -13,13 +13,17 @@ Administrators can create `ClusterLogForwarder` resources that specify which log
 
 Administrators can also authorize RBAC permissions that define which service accounts and users can access and forward which types of logs.
 
-////
-include::modules/log-forwarding-modes.adoc[leveloffset=+1]
+include::modules/log-forwarding-implementations.adoc[leveloffset=+1]
 
-[id="log-forwarding-enabling-multi-clf-mode"]
-== Enabling multi log forwarder mode for a cluster
+[id="log-forwarding-enabling-multi-clf-feature"]
+== Enabling the multi log forwarder feature for a cluster
 
-To use multi log forwarder mode, you must create a service account and cluster role bindings for that service account. You can then reference the service account in the `ClusterLogForwarder` resource to control access permissions.
+To use multi log forwarder feature, you must create a service account and cluster role bindings for that service account. You can then reference the service account in the `ClusterLogForwarder` resource to control access permissions.
+
+[IMPORTANT]
+====
+In order to support multi-cluster log forwarding in additional namespaces other than the `openshift-logging` namespace, you must xref:../../logging/cluster-logging-upgrading.adoc#logging-operator-upgrading-all-ns_cluster-logging-upgrading[update the Cluster Logging Operator to watch all namespaces]. This functionality is supported by default in new Cluster Logging Operator version 5.8 installations.
```

Comment:
nit: AFAICS this is only place we use `multi-cluster log forwarding`, I feel this might cause confusion. 
```suggestion
In order to support multi log forwarding in additional namespaces other than the `openshift-logging` namespace, you must xref:../../logging/cluster-logging-upgrading.adoc#logging-operator-upgrading-all-ns_cluster-logging-upgrading[update the Cluster Logging Operator to watch all namespaces]. This functionality is supported by default in new Cluster Logging Operator version 5.8 installations.
```

Reply:
I'm also fine with this
```suggestion
In order to create `ClusterLogForwarder` resources in additional namespaces other than the `openshift-logging` namespace, you must xref:../../logging/cluster-logging-upgrading.adoc#logging-operator-upgrading-all-ns_cluster-logging-upgrading[update the Cluster Logging Operator to watch all namespaces]. This functionality is supported by default in new Cluster Logging Operator version 5.8 installations.
```

Reply:
It came from @jcantrill 's comments yesterday 😅 but I can change it to multi log forwarding, I think that works better